GitUI/CommandsDialogs/BrowseDialog/GitStatusMonitor.cs

```diff
@@ -309,16 +311,29 @@ private void StartWatchingChanges(string workTreePath, string gitDirPath)
             }
         }
 
-        private void Update()
+        private static bool IsMinimized()
         {
-            ThreadHelper.AssertOnUIThread();
+            if (!EnvUtils.RunningOnWindows())
+            {
+                return false;
+            }
 
-            if (CurrentStatus != GitStatusMonitorState.Running)
+            var currentProcess = Process.GetCurrentProcess();
+            if (currentProcess is null)
             {
-                return;
+                return false;
             }
 
-            if (_nextUpdateTime - Environment.TickCount > 0)
+            return NativeMethods.IsIconic(currentProcess.MainWindowHandle) == Interop.BOOL.TRUE;
```

Comment:
We also have `.IsTrue()` extension
```suggestion
            return NativeMethods.IsIconic(currentProcess.MainWindowHandle).IsTrue();
```